### DISCUSSION TITLE
How do you handle intro edits?

### DISCUSSION THREAD

**Discussion by benpage2** [beetbox/beets#5402] [Q&A]
I've been scratching my brain foe a while on what the best way to handle songs with intro edits would be. 

Getting matches is not the issue here, I know I can just reduce the strength of length in the auto tagger, and use import-repace or another plugin to regex the "(Intro Edit)" in the title away.

I just don't want to loose the "(Intro Edit)" from the title after the import. Sometimes I have the original version and need to be able to differentiate the two.

Any advice is helpful :)

**Comment by RollingStar** [comment]
Can you provide some examples of intro edits?
